### ISSUE TITLE
Training - Tiny Yolo v3 loss function >1 

### ISSUE THREAD

**Issue by jalaldev1980** [pjreddie/darknet#1604]
Hi,
I am training 12k images to detect 5 objects on darknet - Tiny YOLO V3. After 10k iteration the map value was 62% and the average loss function value = 2.02. 

Is the training good ? do I have to bring more images ? why the loss function did not rich to <1?
batch=64
subdivisions=16
width=416
height=416
channels=3
momentum=0.9
decay=0.0005
angle=0

saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.001
burn_in=1000
max_batches = 10000
policy=steps
steps=9000,8000
scales=.1,.1

anchors = 10,14,  23,27,  37,58,  81,82,  135,169,  344,319

**Comment by KatherineNico** [comment]
Hi.  Idk if  you solved your question but I read this:
The final avgerage loss can be from 0.05 (for a small model and easy dataset) to 3.0 (for a big model and a difficult dataset).
From https://github.com/joheras/darknet-1. 
Your models seems to be large indeed so I think it's fine.
